src/modules/job-ingest/validate.c

```diff
@@ -74,13 +74,23 @@ void validate_destroy (struct validate *v)
     }
 }
 
+static inline bool str_ends_with (const char *str, const char *suffix)
+{
+    int str_len = strlen(str);
+    int suffix_len = strlen(suffix);
+
+    return (str_len >= suffix_len) && \
+        (!strncmp((str + str_len) - suffix_len, suffix, suffix_len));
+}
+
```

Comment:
There should be a space between function names and open parens per RFC 7.

While you're in there, I'd probably drop the `inline` for consistency with the rest of the code base and because this is not performance critical code.

Reply:
:+1: